webpack/webpack.base.config.js

```diff
@@ -19,6 +19,14 @@ module.exports = function getBaseConfig() {
     ],
     module: {
       rules: [
+        {
+          test: /\.mjs$/,
+            include: /node_modules/,
+            type: 'javascript/auto',
+            resolve: {
+                fullySpecified: false
+            }
+        },
```

Comment:
Is this also an old fragment? I couldn't find a purpose.

Reply:
Seems to be the case